src/index.html

```diff
@@ -11,7 +11,172 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="information_fieldset">
+        <legend>Personal information</legend>
+
+        <label for="surname" class="personal_information_label"> Surname:
+
+          <input
+            type="text"
+            id="surname"
+            name="surname"
+            autocomplete="off"
+          >
+
+        </label>
+
+        <label for="name" class="personal_information_label"> Name:
+
+          <input
+            type="text"
+            id="name"
+            name="name"
+            autocomplete="off"
+          >
+
+        </label>
+
+        <label for="age" class="personal_information_label"> How old are You?
+
+          <input
+            type="number"
+            name="age"
+            id="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+
+        </label>
+
+        <label for="birthDate" class="personal_information_label"> Full date of birth:
+          <input type="date" id="birthDate">
+        </label>
+
+        <label for="acceptTerms">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            id="acceptTerms"
+            required
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="information_fieldset">
+        <legend>Registration</legend>
+
+        <label for="email" class="registration_fieldset_email">E-mail:
+          <input
+            type="email"
+            id="email"
+            placeholder="email@example.com"
+          >
+        </label>
+
+        <label for="password"> Password:
+          <input
+            type="password"
+            id="password"
+            name="password"
+            minlength="6"
+            maxlength="32"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="information_fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="asking_about_cats_div"> Do you love cats?</div>
```

Comment:
you do not need this div there. You can move `Do you love cats?` to the label below